src/components/ConnectionAwareContainer.vue

```diff
@@ -107,6 +107,10 @@ export default {
       }
     },
     async authenticationGuard() {
+      if (this.$route.meta.skipLogin) {
```

Comment:
Ditto (on props from router)

Reply:
I am sorry, I did not manage to turn this one into a prop.

The problem is this component is loaded by the router and the business component is not aware of it and thus, cannot pass data to it.